templates/OperationResponseExceptionSnippet.stencil

```diff
@@ -1,4 +1,5 @@
 {% for exception in op.exceptions %}
+    {% if exception.statusCodes.count > 0 %}
```

Comment:
Do not generate this code block if the exception is 'default' status code

Reply:
What is default status code? If the logic is based on the default, wouldn't you make the if statement trigger based on `isDefaultStatusCode`?

Reply:
if an exception is default, it will be set to the defaultException in the operationViewModel. And the values will be used by OperationDefaultExceptionSnippet.stenicl.